runtime-sdk/src/module.rs

```diff
@@ -382,9 +382,8 @@ pub trait TransactionHandler {
     /// Perform any action after call, within the transaction context.
     ///
     /// If an error is returned the transaction call fails and updates are rolled back.
-    fn after_handle_call<C: TxContext>(_ctx: &mut C) -> Result<(), modules::core::Error> {
+    fn after_handle_call<C: TxContext>(_ctx: &mut C, _result: &CallResult) {
```

Comment:
So this can no longer abort the call, any reason for removing the return value (I know it was previously not used, but it may be useful in certain cases to be able to do that)? If this is removed the documentation comment above should be updated.